frontend/src/pages/start/Start.tsx

```diff
@@ -1,14 +1,29 @@
 import React, { FunctionComponent } from "react";
-import { Redirect, Route, Switch } from "react-router-dom";
+import { Redirect, Route, RouteProps, Switch } from "react-router-dom";
 import Main from "./main/Main";
 import Configuration from "./configuration/Configuration";
+import Layout from "components/layout/Layout";
+
+const RouteWithLayout = (props: RouteProps) => (
+  <Layout>
+    <Route {...props} />
+  </Layout>
+);
 
 const Start: FunctionComponent = () => {
   return (
     <div>
       <Switch>
         <Route exact path={`/start`} component={Main} />
-        <Route path={`/start/configure/:id?`} component={Configuration} />
+        <Route
+          path={`/start/configure`}
+          exact={true}
+          component={Configuration}
+        />
+        <RouteWithLayout
+          path={`/start/configure/:id?`}
+          component={Configuration}
+        />
```

Comment:
Did you test if the project settings screen logic still works for both of these routes? I think this shouldn't break anything, but just asking to be sure.

Reply:
yep, checked both